prem_utils/connectors/base.py

```diff
@@ -1,10 +1,44 @@
+import json
 from collections.abc import Sequence
 
 import tiktoken
+from openai import (
+    APIConnectionError,
+    APIError,
+    APIResponseValidationError,
+    APIStatusError,
+    APITimeoutError,
+    AuthenticationError,
+    BadRequestError,
+    ConflictError,
+    InternalServerError,
+    NotFoundError,
+    OpenAI,
+    PermissionDeniedError,
+    RateLimitError,
+    UnprocessableEntityError,
+)
 
 from prem_utils import errors
 
 
+def get_provider_blob(provider_name: str) -> dict:
```

Comment:
Why do you need this method?

Reply:
This is the method that parses the json to provide the list of models available by the connector

Reply:
Removed now